apps/web/src/components/Icon/Icon.spec.tsx

```diff
@@ -0,0 +1,8 @@
+export {};
+// import React from 'react';
```

Comment:
Can you add a comment about when this will be updated and link it to an issue? And if we don't think we will write tests for this component this file can be removed.